build/Build.Steps.cs

```diff
@@ -551,12 +556,31 @@ void RemoveFilesFromAdditionalDepsDirectory()
                     AdditionalDepsDirectory.GlobDirectories("**/runtimes").ForEach(DeleteDirectory);
                 }
 
-                string RollFrameworkForward(string runtime, string rollForwardRuntime, IReadOnlyList<string> architectureStores, string depsJsonContent)
+                void RollFrameworkForward(string runtime, string rollForwardRuntime, IReadOnlyList<string> architectureStores, JsonObject jsonDocument)
```

Comment:
nit: static method if possible, if not possible I think it is worth adding a type wrapping the `jsonDocument` to implement these methods.